packages/gestalt-design-tokens/package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "gestalt-design-tokens",
+  "version": "0.1.0",
```

Comment:
@ayeshakmaz this version should be the same as the `gestalt` / `gestalt-datepicker` / ... versions & auto bump when we're doing a release:

https://github.com/pinterest/gestalt/blob/c0c3fd53884c2b4e690f540b04870a4a219c73dc/scripts/releaseSteps.js#L22
https://github.com/pinterest/gestalt/blob/c0c3fd53884c2b4e690f540b04870a4a219c73dc/.github/workflows/release.yml#L86-L91